README.md

```diff
@@ -15,13 +15,27 @@ sudo apt -y install az-dcap-client
 ```
 
 ## Install
+You can use our pre-built binaries to install era on your machine.
+### For the current user
+```bash
+wget -P ~/.local/bin https://github.com/edgelesssys/era/releases/latest/download/era
+chmod +x ~/.local/bin/era
+```
+### Global install (requires root)
+```bash
+sudo -O /usr/local/bin/era https://github.com/edgelesssys/era/releases/latest/download/era
+sudo chmod +x /usr/local/bin/era
+```
 
-1. Download prebuild binaries from our [releases](https://github.com/edgelesssys/era/releases)
-2. Install from source (requires [EdgelessRT](https://github.com/edgelesssys/edgelessrt))
+*Note: On machines running Ubuntu, ~/.local/bin is only added to PATH when the directory exists by default and the user has logged in themselves again since it was created. Also, non-default shells such as `zsh` do not add this path by default. Therefore, if you receive `command not found: era` as an error message for a local user installation, either make sure ~/.local/bin was added to your PATH successfully, or simply use the global installation method.*
```

Comment:
```suggestion
*Note: On machines running Ubuntu, ~/.local/bin is only added to PATH when the directory exists when initializing your bash environment during login. You might need to re-login after creating the directory. Also, non-default shells such as `zsh` do not add this path by default. Therefore, if you receive `command not found: era` as an error message for a local user installation, either make sure ~/.local/bin was added to your PATH successfully or simply use the global installation method.*
```

Reply:
Yup, that sounds better and more descriptive. Will apply.